lib/xebow/rgb_matrix.ex

```diff
@@ -52,31 +59,30 @@ defmodule Xebow.RGBMatrix do
         speed_hz: @spi_speed_hz
       )
 
-    Process.send_after(self(), :tick, @delay_ms)
+    Process.send_after(self(), :run, 0)
+
+    initial_animation = Animations.animations() |> hd()
```

Comment:
This may be out of scope of this PR, but instead of `Animations.animations`, how about something less redundant, like `Animations.get` or `Animations.list`?